tests/aws/services/lambda_/test_lambda_common.py

```diff
@@ -201,12 +220,15 @@ def test_uncaught_exception_invoke(self, multiruntime_lambda, snapshot, aws_clie
         ]
     )
     @markers.aws.validated
-    # this does only work on al2 lambdas, except provided.al2.
+    # Only works for >=al2 runtimes, except for any provided runtimes
     # Source: https://docs.aws.amazon.com/lambda/latest/dg/runtimes-modify.html#runtime-wrapper
     @markers.multiruntime(
         scenario="introspection",
-        runtimes=["nodejs"],
-        # runtimes=["nodejs", "python3.8", "python3.9", "java8.al2", "java11", "dotnet", "ruby"],
+        runtimes=list(
+            set(TESTED_RUNTIMES)
+            - set(RUNTIMES_SKIP_ARM)
```

Comment:
Why do we substract RUNTIMES_SKIP_ARM here? I get it for the al1 runtimes, but currently the list also includes al2, which should work?

Reply:
You are right 🕵️  Newer runtimes should not be subtracted (e.g., >=`java8.al2`, at least not for x86 for now)
That difference came from the added runtimes due to the broken CI builds. We should be able to replace this with `runtimes` by the end of the year.

in any case better than the hardcoded `nodejs` runtimes from before 😅